sources/packages/api/src/route-controllers/student-account-applications/models/student-account-application.dto.ts

```diff
@@ -39,4 +39,6 @@ export class StudentAccountApplicationApprovalAPIInDTO extends CreateStudentAPII
   dateOfBirth: string;
   @Allow()
   gender: string;
+  @Allow()
```

Comment:
This is the Ministry approval and does not seem right to have the student consent received by this endpoint.